Source/JavaScriptCore/dfg/DFGJITCode.cpp

```diff
@@ -64,11 +112,81 @@ JITData::JITData(VM& vm, CodeBlock* codeBlock, const JITCode& jitCode, ExitVecto
             at(i) = codeBlock->globalObject();
             break;
         }
-        default:
+        case LinkerIR::Type::HavingABadTimeWatchpointSet: {
+            auto* globalObject = codeBlock->globalObject();
```

Comment:
Would it be worth pre-caching this `globalObject` outside of this loop?

Reply:
I think we should not do that since in the future we will extend it to accept different JSGlobalObject.